tests/utils.go

```diff
@@ -1340,6 +1340,17 @@ func renderContainerSpec(imgPath string, name string, cmd []string, args []strin
 		Image:   imgPath,
 		Command: cmd,
 		Args:    args,
+		SecurityContext: &k8sv1.SecurityContext{
+			Privileged:               NewBool(false),
+			AllowPrivilegeEscalation: NewBool(false),
+			RunAsNonRoot:             NewBool(true),
```

Comment:
Don't we need to specify user also?

Reply:
Definitely the other tests do not accept this change:
```
"message": "container has runAsNonRoot and image will run as root (pod: \"netcat7v6xv-5mbjs_kubevirt-test-default1(1312c819-9a81-4377-8267-d4c9cf07a3a1)\", container: netcat
```

Reply:
@xpivarc undefined is also fine. https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted

Reply:
@fossedihelm That was my concern
@acardace I believe right now it depends on the image...

Reply:
I have to fix the vm-killer image I think.

Reply:
Was trying to tackle similar issues in https://github.com/kubevirt/kubevirt/pull/8645

Reply:
Done, let's see now.

Reply:
@xpivarc didn't refresh the comments but yes you're right, it should now be fixed.